cmd2/utils.py

```diff
@@ -11,7 +11,7 @@
 import threading
 import unicodedata
 from enum import Enum
-from typing import Any, Callable, Iterable, List, Optional, TextIO, Union
+from typing import Any, Callable, Iterable, List, OrderedDict, Optional, TextIO, Union
```

Comment:
`OrderedDict` was not added to the `typing` module until [Python 3.7.2](https://docs.python.org/3/library/typing.html)

You should be able to directly use [collections.OrderedDict](https://docs.python.org/2/library/collections.html#collections.OrderedDict) in type hints.

This is causing failures on all 3 CI systems.  This is a must fix.